test/suites/integration/validation_test.go

```diff
@@ -194,7 +194,7 @@ var _ = Describe("Validation", func() {
 			nodeClass.Spec.Tags = map[string]string{"karpenter.sh/nodepool": "custom-value"}
 			Expect(env.Client.Create(env.Context, nodeClass)).ToNot(Succeed())
 
-			nodeClass.Spec.Tags = map[string]string{"karpenter.sh/managed-by": env.ClusterName}
+			nodeClass.Spec.Tags = map[string]string{"eks:eks-cluster-name": env.ClusterName}
```

Comment:
Seems odd that we would define the const but not use it here. Any reason why we're not?

Reply:
No reason, updated.